src/Apps/Artwork/Components/ArtworkSidebar2/__tests__/ArtworkSidebar2CommercialButtons.jest.tsx

```diff
@@ -0,0 +1,290 @@
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+import { screen, fireEvent } from "@testing-library/react"
+import { ArtworkSidebar2CommercialButtons_Test_Query } from "__generated__/ArtworkSidebar2CommercialButtons_Test_Query.graphql"
+
+import { mediator } from "Server/mediator"
+import { Toasts, ToastsProvider } from "@artsy/palette"
+import { createMockEnvironment } from "relay-test-utils"
+import { MockBoot } from "DevTools"
+import { ArtworkSidebar2CommercialButtonsFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2CommercialButtons"
+
+jest.unmock("react-relay")
+
+describe("ArtworkSidebar2CommercialButtons", () => {
+  let user
+  beforeAll(() => {
+    mediator.on("open:auth", () => {})
+  })
+
+  let mockEnvironment = createMockEnvironment()
```

Comment:
Curious - was this necessary to pass into `<MockBoot>`?

Reply:
Talked this through with @araujobarret some days ago.

What we wanted to do here was to get access to the mutations that were called after the initial render. The problem was that we were getting the classic error no pending operations in the list. 

After adding the `mockEnvironment` to the MockBoot we were able to locate them with one caveat:
- the initial query was resolved by one environment (the one on setupTestWrapper) but all the rest were resolved by the other one (the one created in the test). 🤯

Also encountered another issue and have it on my list to check it out tomorrow: wasn't able to mock the router in this test.

What I want to do eventually is to migrate all the existing tests from [here](https://github.com/artsy/force/blob/main/src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarCommercial.jest.tsx) to the new spec.

I have it on my list for tomorrow, so any advice is highly appreciated

Reply:
Here's a Q: Is `MockBoot` necessary? Generally we can get away with testing our components in strict isolation, and then mocking out other component deps that might interfere. Is it because of `<Media>` usage? 

Otherwise, this is a really good-to-know. I haven't done too much mutation testing so I can't comment much further but I would continue pushing on as you have! Figuring out all of these quirks and streamlining our testing flows is 👍

Reply:
cc @dzucconi, you might be interested here as well (and might have some ideas)

Reply:
> What we wanted to do here was to get access to the mutations that were called after the initial render.

Not sure why that'd be necessary in a UI spec? I also don't see it in the specs below.